.github/workflows/ci.yml

```
@@ -4,7 +4,7 @@ on:  # yamllint disable-line rule:truthy
   # Trigger the workflow on pushes to the main branch and all pull requests.
   push:
     branches:
-      - main
+      - 2.x
```

```suggestion
      - *.x
```
Would this work?

---

[I think so](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore) ([1](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)). Either that or 
```suggestion
      - **.x
```

---

I think your suggestion is correct. ✅